capture/noworkflow/now/vis/views.py

```diff
@@ -336,7 +338,11 @@ def get_file(file_hash, file_ext):
 @app.route("/trials/<tid>/<graph_mode>/<cache>.json")
 def trial_graph(tid, graph_mode, cache,expCode=None):
     """Respond trial graph as JSON"""
-    trial = Trial(tid)
+    if "definition" in graph_mode :
+        trial = Definition(tid)
+        graph_mode = graph_mode.split('_')[1]
+    else:
+        trial = Trial(tid)
```

Comment:
If we had Definition as a graph_mode of Trial as suggested before, this wouldn't be required.